noir-projects/aztec-nr/aztec/src/oracle/mod.nr

```diff
@@ -2,24 +2,25 @@
 /// Oracles module
 // docs:end:oracles-module
 
+pub mod aes128_decrypt;
```

Comment:
alphabetising

Reply:
Just FYI. The order in mod or lib files determines the order in which comptime code is processed and this sometimes need to be in a specific way. For that reason I had to re-order the modules in protocol circuits in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/11774).

Reply:
Oh interesting. Since CI seems to be happy, does that mean my reordering is acceptable at the moment?

Reply:
My PR is not merged and it was focused on protocol circuits. I might just need to break the ordering here later on as we derive more trait implementations